packages/addons/service/mpd/source/resources/language/English/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Probably best left for a follow up PR. I'm only copying this as-is because it's been requested by users and is currently missing, I definitely don't want to become the package maintainer...